sktime/utils/plotting.py

```diff
@@ -109,3 +109,70 @@ def format_fn(tick_val, tick_pos):
         ax.legend()
 
     return fig, ax
+
+
+def plot_lags(series, lags=1):
+    """Plot one or more lagged versions of a time series
```

Comment:
Will need to make minor formatting changes to the docstring now that sktime has turned on pydocstyle docstring code quality checks.